weblate/static/editor/full.js

```diff
@@ -578,7 +578,7 @@
           .attr("dir", this.state.dir)
           .html(el.html),
       );
-      row.append($("<td>").html(el.diff));
+      row.append($("<td>").attr("class", "machinery-diff").html(el.diff));
```

Comment:
another approach could be to simply add the already defined `machinery-text` class which uses the rule `word-wrap: anywhere`, and has pretty much the same effect

Reply:
That sounds even better. There is no reason to apply different logic to diff than to the full text.